hunt/app/views/animals/index.html.erb

```diff
@@ -1,7 +1,44 @@
  <div class="container">
    <div class="row">
      <div class="col">
-       <%= link_to 'New Animal', new_animal_path, class: 'btn btn-dark' %>
+
+
+         <div class="card-header">
```

Comment:
Generally a card header is really only used for some text and/or buttons. Lets use the `card-body` for this instead. If you wanted this for the gray color we have some bootstrap utility classes we can use. `bg-light` should give it a light gray background. This also needs to be wrapped in a `<div class="card">` like on the other pages. So in total that would be:

```
<div class="card">
  <div class="card-body bg-light">
    // All your form/button stuff here
  </div>
</div>
```

Besides that, we just need to clean up the indentation and extra blank lines and we should be good!!